ui/app/src/views/auth/SignWrapper.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { Container, Divider, useMediaQuery, useTheme } from '@mui/material';
+import { useDarkMode } from '../../context/DarkMode';
+import PersesLogoCropped from '../../components/logo/PersesLogoCropped';
+import DarkThemePersesLogo from '../../components/logo/DarkThemePersesLogo';
+import LightThemePersesLogo from '../../components/logo/LightThemePersesLogo';
+
+export function SignWrapper(props: { children: React.ReactNode }) {
+  const { isDarkModeEnabled } = useDarkMode();
+  const isLaptopSize = useMediaQuery(useTheme().breakpoints.up('sm'));
+  return (
+    <Container
+      maxWidth="sm"
+      sx={{
+        display: 'flex',
```

Comment:
<Stack> is more adapted flex displays

Reply:
I tried

```jsx
    <Stack
      direction={{ xs: 'column', sm: 'row' }}
      justifyContent="center"
      alignItems="center"
      maxWidth="sm"
      divider={<Divider orientation={isLaptopSize ? 'vertical' : 'horizontal'} flexItem />}
      spacing={2}
    >
```

But it doesn't seem the `justifyContent` is working it positions the element on the left like if `justifyContent` was equal to `flex-start` :(